megamek/src/megamek/client/ui/swing/ChatLounge.java

```diff
@@ -3483,7 +3483,7 @@ public Component getTableCellRendererComponent(JTable table, Object value, boole
                         if (compact) {
                             clearImage();
                         } else {
-                            setPortrait(entity.getCrew());
+                            setImage(entity.getCrew().getPortrait(0).getImage(50));
```

Comment:
Those are oddly specific values. Should they be named?

Reply:
We don't name sizes currently, so I would say not

Reply:
Wasn't obvious that was the square size, makes sense now.